content/cloud-docs/workspaces/variables/managing-variables.mdx

```diff
@@ -9,17 +9,20 @@ description: >-
 
 You can set variables specifically for each workspace or you can create variable sets to reuse the same variables across multiple workspaces. Refer to the [variables overview](/cloud-docs/workspaces/variables) documentation for more information about variable types, scope, and precedence. You can also set variable values specifically for each run on the command line.
 
+You can create and edit workspace-specific variables through:
```

Comment:
@laurapacilio I moved the create/edit workspace specific variables section here. Making sure this is actually the right place!

Reply:
yes, thank you!